x-pack/plugins/index_management/server/test/helpers/route_dependencies.ts

```diff
@@ -15,6 +15,7 @@ export const routeDependencies: Omit<RouteDependencies, 'router'> = {
     isLegacyTemplatesEnabled: true,
     isIndexStatsEnabled: true,
     isDataStreamsStorageColumnEnabled: true,
+    enableTogglingDataRetention: true,
```

Comment:
ℹ️ this is irrelevant with my changes but it failed in this PR. It also fails in main, I am not sure why the CI check hasnt marked it before. Possibly because it is on server side (?)